src/components/common/topicsFilter/index.tsx

```diff
@@ -272,10 +272,12 @@ const CreateTopic = ({ onCreateCamp = () => {} }) => {
               </Title>
               <Popover content="Algorithm Information" placement="top">
                 {router.asPath.includes("/topic") ? (
-                  <a href={K?.Network?.URL?.helpTopicUrl}>Help</a>
+                  <a href={K?.Network?.URL?.algoInfoUrl}>
```

Comment:
Aren't we using Link?

Reply:
Yes, But in this case we need to reload the page